SETUP.md

```diff
@@ -157,6 +185,35 @@ If you continue to have issues with rmagick, after changing your imagemagick ver
 - `gem uninstall rmagick`
 - `gem install rmagick -v 2.15.4`
 
+#### libv8
+
+If you run into an error with libv8 while running bundle install
+- Uninstall libv8: `gem uninstall libv8`
+- Make sure the gem no longer exists with: `gem list libv8`
+- Install the current version used in code.org repo: `gem install libv8 -v CURRENT_CODEORG_VERSION -- --with-system-v8` (you can find the current version in the [Gemfile.lock](./Gemfile.lock)).
+
+#### mysql2
+
+If you run into an issue about mysql2 while running `bundle install` and the error output includes "ld: library not found for -lssl" try :
+- `brew install openssl`
+- `export LIBRARY_PATH=$LIBRARY_PATH:/usr/local/opt/openssl/lib/`
+
+(Steps from [this github issue](https://github.com/brianmario/mysql2/issues/795#issuecomment-439579677))
+
+#### therubyracer
+
+If you run into an issue about therubyracer while running `bundle install` try :
+-`gem uninstall libv8`
```

Comment:
Add a space between the - and the ` i.e.

Without space:

-`gem uninstall libv8`

with space
- `gem uninstall libv8`

Reply:
Updated!